lib/rules/web/admin_console/4.5/clustersettings.xyaml

```diff
@@ -0,0 +1,26 @@
+click_update_channel_button:
+  elements:
+  - selector:
+      xpath: //button[@data-test-id='current-channel-update-link']
+    op: click
+click_channels_drop_down:
+  elements:
+  - selector:
+      xpath: //button[@data-test-id='dropdown-button']
+    op: click
```

Comment:
We have a same rule with same definition but a different name
```
click_dropdown_button:
  element:
    selector:
      xpath: //button[contains(@data-test-id, 'dropdown-button')]
    op: click
```

Reply:
@yapei update:
- removed the redundant rule and used the existing web rule as suggested.